utils/merkletrie/filesystem/node_test.go

```diff
@@ -196,6 +199,24 @@ func (s *NoderSuite) TestDiffDirectory(c *C) {
 	c.Assert(a, Equals, merkletrie.Modify)
 }
 
+func (s *NoderSuite) TestSocket(c *C) {
+	td, err := os.MkdirTemp("", "socket-test")
```

Comment:
@steiler please skip this test on Windows - which is why the tests are failing at the moment.

Reply:
yes, done